src/frontend/packages/suse-extensions/src/custom/kubernetes/kubernetes-resource-viewer/kubernetes-resource-viewer.component.ts

```diff
@@ -44,32 +45,41 @@ export class KubernetesResourceViewerComponent implements PreviewableComponent {
   public hasPodMetrics$: Observable<boolean>;
   public podRouterLink$: Observable<string[]>;
 
+  private analysis;
+  private alerts;
+
   setProps(props: KubernetesResourceViewerConfig) {
     this.title = props.title;
+    this.analysis = props.analysis;
```

Comment:
This is probably a v2 thing, but would be cool to tie this into the pods lists as well.
- They should all use a common base config service (`BaseKubernetesPodsListConfigService`)
- Would need an extra column to show some kind of summary state
- Would need `analysis` property passed through in name cell's cellConfig 
- The overlay enable/selection drop down integration might be tricky. Maybe the kube analysis service could store this and be provided/have an instance per place it's used?